core/src/main/java/org/vertexium/type/GeoPoint.java

```diff
@@ -92,6 +118,7 @@ public int hashCode() {
         hash = 47 * hash + (int) (Double.doubleToLongBits(this.latitude) ^ (Double.doubleToLongBits(this.latitude) >>> 32));
         hash = 47 * hash + (int) (Double.doubleToLongBits(this.longitude) ^ (Double.doubleToLongBits(this.longitude) >>> 32));
         hash = 47 * hash + (this.altitude != null ? this.altitude.hashCode() : 0);
+        hash = this.accuracy != null ? (47 * hash + this.accuracy.hashCode()) : 0;
```

Comment:
This looks like a typo. If no accuracy is provided, the hash will always be 0

Reply:
Wow, how did you catch that?

Reply:
Reading through the changes, it caught my attention that it didn't look like the other lines above it :-)